docs/source/conf.py

```diff
@@ -417,6 +433,7 @@ def generate_weights_table(module, table_name, metrics, dataset, include_pattern
     table_name="classification_quant",
     metrics=[("acc@1", "Acc@1"), ("acc@5", "Acc@5")],
     dataset="ImageNet-1K",
+    exclude_columns=["GFLOPs", "Size (MB)"],
```

Comment:
The size in MB for quantized model should be displayed. It's only the flops that we can't current estimate.

Reply:
True, these slipped.